### PR TITLE
ADD new style to the index.html page

### PR DESCRIPTION
**Pull request by enoqueJonas**
Added new:
 - Style-wide-screen.css
 - Background pictures for #header and #contact

This .css file affects the index.html page. It has a style that will be loaded on a big screen (768px wide)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -167,7 +173,7 @@ <h2 class="works-item-title"> Tonic </h2>
                 </div>
             </div>
             <div class="works-item works-item-4">
-                <div class="works-item-image-4"></div>
+                <div class="works-item-image-4 works-item-image"></div>
                 <div class="works-item-info-block">
                     <div class="works-item-project-info">
```
Comment (ghost) [inline index.html:178]: - [ ] Kindly space your contents in all your working cards to match the Figma design. The image and the description are very close

image

image

### Checkout Figma

image

- [ ] Kindly align the contents of your `about me` section to the center of the page to match the Figma design.

image

- [ ] Kindly note that there are two lines separating frameworks and skills.
image

### Checkout figma

image

- [ ] Kindly align the text in your `get in touch` button to the center to match the Figma design

image

### Checkout Figma

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @enoqueJonas , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

YesYouAreGIF (2)


## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style-wider-screen.css [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+#about {
+  justify-content: center;
+  height: 724px;
+  left: 1440px;
+  background: #fff;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+}
+
```
Comment (jerryowusu) [inline css/style-wider-screen.css:177]: [OPTIONAL] At screen size between 768px and 1150px, part of the about section pushes to the left side and goes missing. This does not make your project smooth and responsive. So kindly fix that. :pray: 


**At screen width 816px**

Screenshot from 2022-09-30 16-52-04

**At screen width 1074px**

Screenshot from 2022-09-30 16-50-28

**Review by jerryowusu — APPROVED** [review]
Hi @enoqueJonas 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

good job


## Highlights
All linters are passing :heavy_check_mark:
Correct GitHub workflow :heavy_check_mark:
Meaningful commit messages :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**